src/model/implementation/root-entity-type.ts

```diff
@@ -61,6 +61,18 @@ export class RootEntityType extends ObjectTypeBase {
         return indices.filter((index, i1) => !indices.some((other, i2) => i1 < i2 && other.equals(index)));
     }
 
+    @memorize()
+    get arangoSearchConfig(): ArangoSearchIndexConfig {
```

Comment:
the term `arangoSearch` should not occur in the general model classes (also applies to `input`). Guess should just be `flexSearch` here.